pkg/image/origins.go

```diff
@@ -154,6 +160,8 @@ func init() {
 		"mirrored-longhornio-longhorn-ui":                         "https://github.com/longhorn/longhorn-ui",
 		"mirrored-messagebird-sachet":                             "https://github.com/messagebird/sachet",
 		"mirrored-metrics-server":                                 "https://github.com/kubernetes-sigs/metrics-server",
+		"mirrored-minio-mc":                                       "https://github.com/minio/mc",
```

Comment:
Is it feasible to add a check to `convertImagesToRepoUrls` to check for this the same way we do for `hardened`? Seems mirrored images could potentially check `rancher/<REPO>`, then `<ORG>/<REPO>` before setting as unknown.

Reply:
Good idea, added in latest commit

Reply:
I added that logic in to the latest commits 👍